python/gaas_client/gaas_thrift.py

```diff
@@ -57,11 +57,17 @@
   3:list<i32> path_sizes
 }
 
-union DataframeRowIndex {
-  1:i32 int32_index
-  2:i64 int64_index
-  3:list<i32> int32_indices
-  4:list<i64> int64_indices
+struct UniformNeighborSampleResult {
+  1:list<i32> sources
+  2:list<i32> destinations
+  3:list<double> indices
```

Comment:
`uniform_neighbor_sample` in cugraph will return the indices with a dtype based on the weights of the incoming graph. I believe this is intentional so that weights can optionally be used for edge IDs, but I think this will be changing once we add support for handling edge IDs and type enums at the PLC and C layers.  I've added a FIXME and [this issue](https://github.com/rapidsai/cugraph/issues/2654) to track it for now.